### PR TITLE
New York Times website

### PR DESCRIPTION
**Pull request by KartheeKWD**
We have used grid and flex to make a new york times website clone

### CODE REVIEW

**Comment by sumancrest0001** [comment]
**Status: changes required 
Nicely done TEAM :clap: :+1:** 
You have implemented most of the requirements from the project specifications but a few minor changes are necessary to get approved. Let's make the following changes

[NOTE] optional changes are not mandatory but nice to have.
1. We have to make the NYT logo on the navbar as a link. Let's change the subscribe now link into a button to give it similar looks like that of the original one. We can give some padding to the navbar on the x-axis.
in your project :point_down: 
Screenshot from 2020-06-17 16-30-17

original one :point_down: 

Screenshot from 2020-06-17 16-30-33

2. The sticky footer section is missing for the page layout. It looks like this :point_down:  on the original page.
Screenshot from 2020-06-17 16-26-10

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Please request another code review after making the requested changes.
If you have any questions or need help, DM me on slack Suman Shrestha
Happy coding !!!**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+  grid-row: 9 / 13;
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: 100px 125px;
+  grid-template-rows: 50px 80px 80px 80px 80px;
+  grid-row-gap: 10px;
+  margin-left: 50px;
+  margin-top: 100px;
+  float: right;
+}
```
Comment (trekab) [inline style.css]: This `float` rule has no effect on the page.

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required! :recycle: 
Hi @KartheekWD and @defoebrand, you have done an impressive job using flex and grid on this project. However, there is one requirement you need to work on to complete this project.
- [ ] Elements should be aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction)). Your current use of float has no visual effect on the page.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by addod19** [comment]
## Pull  Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi  **TEAM**, great job with fixing all the comments made by the previous TSE's :+1: Thanks to your persistent desire to learn and improve, and the help of our TSE, our project now meets all the minimum requirements :smile: 

Go ahead and merge to master and submit project completion form :+1: 
Good luck with your next projects :100: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**